modules/displayList.js

```diff
@@ -0,0 +1,69 @@
+export default class TaskList {
+  constructor() {
+    this.getTasksFromLocalStorage();
+  }
+
+  updateTasksInLocalStorage() {
+    localStorage.setItem('taskList', JSON.stringify(this.listArray));
+  }
+
+  getTasksFromLocalStorage() {
+    this.listArray = JSON.parse(localStorage.getItem('taskList')) || [];
+  }
```

Comment:
Kindly use the ES6 arrow function instead of normal functions. To make your code clean.

Reply:
Thankyou Nahom for the amazing feedback 👍